### PR TITLE
[Frozen] Soups tweaks

### PR DESCRIPTION
**Pull request by lapatison**
## About the PR

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added several soup recipes
- add: Added soup flavors

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml [hunk 1]
```diff
@@ -378,7 +369,8 @@
           Quantity: 4
         - ReagentId: Vitamin
           Quantity: 4
-# Tastes like rice, egg. Egg =/= Meat -Swept 2020
+# Egg =/= Meat -Swept 2020
```
Comment (moonheart08) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:372]:
```suggestion

```

**Comment by deltanedas** [comment]
add; -> add:

**Comment by lapatison** [comment]
> add; -> add:

Done

**Comment by metalgearsloth** [comment]
As per the others when they use cgraphs re-open.

**Comment by keronshb** [comment]
@lapatison you didn't have to delete the branch since you could reopen this once the chef rework is in

**Comment by lapatison** [comment]
> @lapatison you didn't have to delete the branch since you could reopen this once the chef rework is in

Crap, I forgot that renaming branch deletes it.
Ill restore it back and will wait for cooking update

**Comment by metalgearsloth** [comment]
Under https://github.com/space-wizards/space-station-14/issues/8524
As food doesn't really have a purpose it's subject to removal at any time pending chef work.

**Pull request merged**